File: JavaScript/Busqueda_binaria.js
let arreglo = [3, 8, 15, 23, 42, 56, 70];

function busBinaria(valor) {
    let izquierda = 0;
    let derecha = arreglo.length - 1;

    while (izquierda <= derecha) {
        let medio = Math.floor((izquierda + derecha) / 2);

        if (arreglo[medio] === valor) {
            return medio;
        } else if (arreglo[medio] < valor) {
            izquierda = medio + 1;
        } else {
            derecha = medio - 1;
        }
    }
    return -1;
}

console.log(busBinaria(23));

File: Python/Busqueda_binaria.py
arreglo = [2, 6, 23, 45, 67, 88]

def bus_bin(valor):
    izquierda = 0
    derecha = len(arreglo) - 1
    while izquierda <= derecha:
        medio = (izquierda + derecha ) // 2
        if arreglo[medio] == valor:
            return medio 
        elif arreglo[medio] < valor:
            izquierda = medio + 1
        else:
            derecha = medio - 1
    return -1

print(bus_bin(88))